frontend/test/__runner__/backend.js

```diff
@@ -8,15 +8,15 @@ const { spawn } = require("child_process");
 const fetch = require("isomorphic-fetch");
 
 let testDbId = 0;
-const getDbFile = () =>
+const generateTempDbFile = () =>
   path.join(os.tmpdir(), `metabase-test-${process.pid}-${testDbId++}.db`);
```

Comment:
Does it actually generate a DB?

Reply:
More precisely, it generates a string/path that is then passed to the `MB_DB_FILE` and is created during a `spawn` process. What would be better name for this? I'm blanking out.

Reply:
WDYT about something like `generateTempDbFilePath` or `getTempDbPath`? Does it sound / represents the idea better?